multinet/api.py

```diff
@@ -2,9 +2,11 @@
 from flasgger import swag_from
 from flask import Blueprint, request
 from webargs import fields
-from webargs.flaskparser import use_kwargs
 
-from typing import Any, Optional
+# NOTE: Not sure why mypy flags this
+from webargs.flaskparser import use_args, use_kwargs  # type: ignore
```

Comment:
The reason is we're using our own type stubs to provide types for this module; see mypy_stubs/webargs/flaskparser.pyi for where to fix this error.

Reply:
However, before fixing this, let's figure out if we can leave the `get_workspace_tables()` function more or less as it was before, which would obviate the need to use `use_args()` anyway.

Reply:
This will become moot because of how we're fixing the problem of a parameter `type` in `get_workspace_tables()`.